src/connected-components/TextArea/TextArea.example.st.css

```diff
@@ -0,0 +1,19 @@
+
+:import {
+  -st-from: "wix-ui-tpa/TextArea/TextArea.st.css";
+  -st-default: TPAMixin;
+}
+
+.root {
+  -st-mixin: TPAMixin(
+      BorderColor --TextArea-BorderColor,
+      BackgroundColor --TextArea-BackgroundColor,
+      TextColor --TextArea-TextColor,
+      DisabledTextColor --TextArea-DisabledTextColor,
+      PlaceholderColor --TextArea-PlaceholderColor,
+      TextFont --TextArea-TextFont,
+      ErrorBorderColor --TextArea-ErrorBorderColor,
+      SuccessBorderColor --TextArea-SuccessBorderColor,
+      DisabledBorderColor --TextArea-DisabledBorderColor
+  );
+}
```

Comment:
I'd rather have this file inside the `TextArea` component folder, instead of adding the `connected-components` directory.
In addition, I think we already generate a `${componentName}ExtendedExample` file in the component's `docs` folder.
Maybe we can use this file instead of creating a new one?